spec/models/publications/monthly_statistics/by_age_group_spec.rb

```diff
@@ -28,4 +31,13 @@
 
     expect_column_totals(4, 2, 1, 1, 1, 7, 16)
   end
+
+  def generate_deleted_application
+    deleted_candidate = create_and_advance(:candidate, hide_in_reporting: false, email_address: 'deleted-application-gh1111@example.com')
+    deleted_application = create_and_advance(:application_form, date_of_birth: nil, candidate: deleted_candidate)
+    create_and_advance(:application_choice,
+                       :with_rejection,
+                       course_option: course_option_with(level: 'primary', program_type: 'higher_education_programme', region: 'eastern', subjects: [primary_subject(:mathematics)]),
+                       application_form: deleted_application)
+  end
```

Comment:
if you have time (can be done as a separate PR later if you like) would be good to have an explicit assertion around this.

Reply:
You mean asserting the generated data? On this spec is already doing this :)

Reply:
do we have an assertion that explicitly states "the report does not include the deleted application"?